src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -67,4 +80,20 @@ public void patchObjection(String requestId, String companyNumber, String object
         }
     }
 
+    @Override
+    public ServiceResult<String> addAttachment(String objectionId, MultipartFile file) throws ServiceException {
```

Comment:
Sorry if it seems I keep banging on about it, you have passed the objectionId from the controller into this method here, but are using the objetionId as the requestId argument in fileTransferApiClient.upload() on line 85.
I think you should be passing the requestId in as this param and then passing the requestId into the fileTransferApiClient.upload()

Reply:
maybe rename requestId to httpRequestId to clear it up as in Extensions the Request was the entity, but now Objection is the entity.

Reply:
Renaming the requestId a good idea but I'd rather put it to one side for now as it relates to several areas of the code. I'll stick with requestId for now.

Reply:
Changed this.